rclcpp/include/rclcpp/strategies/allocator_memory_strategy.hpp

```diff
@@ -195,6 +194,14 @@ class AllocatorMemoryStrategy : public memory_strategy::MemoryStrategy
     return has_invalid_weak_nodes;
   }
 
+  void add_waitable_handle(const rclcpp::Waitable::SharedPtr & waitable)
```

Comment:
@wjwwood  this line introduced a build warning on clang https://ci.ros2.org/view/nightly/job/nightly_osx_debug/1577/clang/new/

Reply:
Oh yes, it's missing the `override` keyworkd

Reply:
I'm looking into it.

Reply:
https://github.com/ros2/rclcpp/pull/1040

Reply:
I merged the fix just now, sorry for the disruption.